<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>301</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>471</width>
    <height>301</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>471</width>
    <height>301</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FractIOns</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/math.png</normaloff>images/math.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #f6e6cb;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_chisl1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
color: #a0937d;
border: 1px solid #a0937d;
  qproperty-alignment: 'AlignVCenter | AlignCenter';
border-radius: 4px;
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_znam1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
color: #a0937d;
border: 1px solid #a0937d;
  qproperty-alignment: 'AlignVCenter | AlignCenter';
border-radius: 4px;
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_chisl2">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>40</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
color: #a0937d;
border: 1px solid #a0937d;
  qproperty-alignment: 'AlignVCenter | AlignCenter';
border-radius: 4px;
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_znam2">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>100</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
color: #a0937d;
border: 1px solid #a0937d;
  qproperty-alignment: 'AlignVCenter | AlignCenter';
border-radius: 4px;
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_cel2">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>80</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
color: #a0937d;
border: 1px solid #a0937d;
  qproperty-alignment: 'AlignVCenter | AlignCenter';
border-radius: 4px;
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_cel1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
color: #a0937d;
border: 1px solid #a0937d;
  qproperty-alignment: 'AlignVCenter | AlignCenter';
border-radius: 4px;
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_chisl">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
color: #a0937d;
  qproperty-alignment: 'AlignVCenter | AlignCenter';
border-radius: 4px;
}

background-color : white;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_znam">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>100</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
color: #a0937d;
  qproperty-alignment: 'AlignVCenter | AlignCenter';
border-radius: 4px;
}

background-color : white;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>80</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
color: #a0937d;
  qproperty-alignment: 'AlignVCenter | AlignCenter';
border-radius: 4px;
}

background-color : white;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AC">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>160</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color: #a0937d;
border: 2px solid #a0937d;
border-radius: 8px;
background-color: #e3cdc1;

cursor: pointer;

}
QPushButton:hover {
transition: 0.9s;
background-color: faf2da;
color: #ededf9;
}
</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>160</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color: #a0937d;
border: 2px solid #a0937d;
border-radius: 8px;
background-color: #e3cdc1;

cursor: pointer;

}
QPushButton:hover {
transition: 0.9s;
background-color: faf2da;
color: #ededf9;
}
</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ymnojenie">
    <property name="geometry">
     <rect>
      <x>283</x>
      <y>160</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color: #a0937d;
border: 2px solid #a0937d;
border-radius: 8px;
background-color: #e3cdc1;

cursor: pointer;

}
QPushButton:hover {
transition: 0.9s;
background-color: faf2da;
color: #ededf9;
}
</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>189</x>
      <y>160</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color: #a0937d;
border: 2px solid #a0937d;
border-radius: 8px;
background-color: #e3cdc1;

cursor: pointer;

}
QPushButton:hover {
transition: 0.9s;
background-color: faf2da;
color: #ededf9;
}
</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_delenie">
    <property name="geometry">
     <rect>
      <x>377</x>
      <y>160</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color: #a0937d;
border: 2px solid #a0937d;
border-radius: 8px;
background-color: #e3cdc1;

cursor: pointer;

}
QPushButton:hover {
transition: 0.9s;
background-color: faf2da;
color: #ededf9;
}
</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_obmen">
    <property name="geometry">
     <rect>
      <x>283</x>
      <y>230</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color: #a0937d;
border: 2px solid #a0937d;
border-radius: 8px;
background-color: #e3cdc1;

cursor: pointer;

}
QPushButton:hover {
transition: 0.9s;
background-color: faf2da;
color: #ededf9;
}
</string>
    </property>
    <property name="text">
     <string>@</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stepen">
    <property name="geometry">
     <rect>
      <x>189</x>
      <y>230</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color: #a0937d;
border: 2px solid #a0937d;
border-radius: 8px;
background-color: #e3cdc1;

cursor: pointer;

}
QPushButton:hover {
transition: 0.9s;
background-color: faf2da;
color: #ededf9;
}
</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_smena">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>230</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color: #a0937d;
border: 2px solid #a0937d;
border-radius: 8px;
background-color: #e3cdc1;

cursor: pointer;

}
QPushButton:hover {
transition: 0.9s;
background-color: faf2da;
color: #ededf9;
}
</string>
    </property>
    <property name="text">
     <string>+-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_otmena">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>230</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color: #a0937d;
border: 2px solid #a0937d;
border-radius: 8px;
background-color: #e3cdc1;

cursor: pointer;

}
QPushButton:hover {
transition: 0.9s;
background-color: faf2da;
color: #ededf9;
}
</string>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sokrat">
    <property name="geometry">
     <rect>
      <x>377</x>
      <y>230</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color: #a0937d;
border: 2px solid #a0937d;
border-radius: 8px;
background-color: #e3cdc1;

cursor: pointer;

}
QPushButton:hover {
transition: 0.9s;
background-color: faf2da;
color: #ededf9;
}
</string>
    </property>
    <property name="text">
     <string>~</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_znak">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>78</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
color: #a0937d;
  qproperty-alignment: 'AlignVCenter | AlignCenter';
border-radius: 4px;
}

background-color : white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_stepen">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>34</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
color: #a0937d;
  qproperty-alignment: 'AlignVCenter | AlignCenter';
border-radius: 4px;
}

</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_stepen">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>18</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
color: #a0937d;
border: 1px solid #a0937d;
  qproperty-alignment: 'AlignVCenter | AlignCenter';
border-radius: 4px;
}
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ravno">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>75</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color: #a0937d;
border: 2px solid #a0937d;
border-radius: 8px;
background-color: #e3cdc1;

cursor: pointer;

}
QPushButton:hover {
transition: 0.9s;
background-color: faf2da;
color: #ededf9;
}
</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_black1">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>93</y>
      <width>61</width>
      <height>5</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color : #a0937d;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_black2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>93</y>
      <width>61</width>
      <height>5</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color : #a0937d;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_black3">
    <property name="geometry">
     <rect>
      <x>385</x>
      <y>93</y>
      <width>61</width>
      <height>5</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color : #a0937d;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_stepen_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>234</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
color: #a0937d;
border: 1px solid #a0937d;
  qproperty-alignment: 'AlignVCenter | AlignCenter';
border-radius: 4px;
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_stepen_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
color: #a0937d;
  qproperty-alignment: 'AlignVCenter | AlignCenter';
border-radius: 4px;
}

</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <zorder>lineEdit_stepen_2</zorder>
   <zorder>label_stepen_2</zorder>
   <zorder>label_chisl</zorder>
   <zorder>label_znam</zorder>
   <zorder>label_cel</zorder>
   <zorder>pushButton_AC</zorder>
   <zorder>pushButton_plus</zorder>
   <zorder>pushButton_ymnojenie</zorder>
   <zorder>pushButton_minus</zorder>
   <zorder>pushButton_delenie</zorder>
   <zorder>pushButton_obmen</zorder>
   <zorder>pushButton_stepen</zorder>
   <zorder>pushButton_smena</zorder>
   <zorder>pushButton_otmena</zorder>
   <zorder>pushButton_sokrat</zorder>
   <zorder>pushButton_ravno</zorder>
   <zorder>label_black3</zorder>
   <zorder>lineEdit_chisl2</zorder>
   <zorder>lineEdit_stepen</zorder>
   <zorder>lineEdit_chisl1</zorder>
   <zorder>lineEdit_znam1</zorder>
   <zorder>lineEdit_cel1</zorder>
   <zorder>label_znak</zorder>
   <zorder>lineEdit_cel2</zorder>
   <zorder>label_black2</zorder>
   <zorder>label_stepen</zorder>
   <zorder>lineEdit_znam2</zorder>
   <zorder>label_black1</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
